src/main/java/core/basesyntax/service/ParceData.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.modelfruit.ModelFruit;
+import java.util.List;
+
+public interface ParceData {
```

Comment:
```suggestion
public interface DataParser {
```
![image](https://user-images.githubusercontent.com/96740815/174764052-4d893b5d-abb1-4465-a27c-79aea0903719.png)
Перевір інші назви класів